<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_5">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_6">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>160</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_8">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>300</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_9">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>300</y>
      <width>120</width>
      <height>120</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
      <italic>false</italic>
      <underline>true</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>?</string>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStart">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>401</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>601</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true"/>
    </property>
    <property name="pixmap">
     <pixmap>../MVil3K0a_4x.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>btn_1</zorder>
   <zorder>btn_3</zorder>
   <zorder>btn_2</zorder>
   <zorder>btn_5</zorder>
   <zorder>btn_4</zorder>
   <zorder>btn_6</zorder>
   <zorder>btn_8</zorder>
   <zorder>btn_7</zorder>
   <zorder>btn_9</zorder>
   <zorder>btnStart</zorder>
   <zorder>output</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
